src/main/java/frc/robot/commands/AngleHood.java

```diff
@@ -0,0 +1,51 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotMap.HoodMap;
+import frc.robot.subsystems.Hood;
+
+public class AngleHood extends CommandBase {
+  /** Creates a new AngleHood. */
+  Hood hood;
+  int MotorDirection;
+
+  public AngleHood(Hood p_hood, int p_MotorDirection) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    hood = p_hood;
+    MotorDirection = p_MotorDirection;
+    addRequirements(hood);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    // Start Spinning hood motor
+    // Set Angle
+    hood.setAngle(MotorDirection * .2);
```

Comment:
Do this multiplication on the method itself. You always want to do math at the lowest level.